Index: include/wx/msw/wrapwin.h
===================================================================
--- include/wx/msw/wrapwin.h	(revision 62325)
+++ include/wx/msw/wrapwin.h	(working copy)
@@ -35,7 +35,7 @@
     // expected both with standard VC6 headers (which define WINVER as 0x0400
     // by default) and headers from a newer SDK (which may define it as 0x0500)
     #if !defined(__VISUALC__) || (__VISUALC__ >= 1300)
-        #define WINVER 0x0600
+        #define WINVER 0x0501
     #endif
 #endif
 
@@ -45,7 +45,7 @@
 // doesn't want to use APIs only available on later OS versions and had defined
 // them to (presumably lower) values
 #ifndef _WIN32_WINNT
-    #define _WIN32_WINNT 0x0600
+    #define _WIN32_WINNT 0x0501
 #endif
 
 #ifndef _WIN32_IE
